paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncDatabaseAction.java

```diff
@@ -143,7 +143,8 @@ protected void beforeBuildingSourceSink() throws Exception {
         for (JdbcTableInfo tableInfo : jdbcTableInfos) {
             Identifier identifier =
                     Identifier.create(
-                            database, tableNameConverter.convert(tableInfo.toPaimonTableName()));
+                            database,
+                            tableNameConverter.convert("", tableInfo.toPaimonTableName()));
```

Comment:
database?

Reply:
Actually, I changed nothing but supply default value for the first argument for the "convert" method. 
For this context, the prefix and suffix are not specified by database, so it's ok to provide an empty string as the argument value.